airgun/views/all_hosts.py

```diff
@@ -449,3 +449,94 @@ class review(WizardStepView):
     @property
     def is_displayed(self):
         return self.browser.wait_for_element(self.title, exception=False) is not None
+
+
+class ManageRepositorySetsModal(Modal):
+    """
+    This class represents the Manage Repository Sets modal that is used to apply content overrides actions
+    on one or more repository.
+    It contains several nested views that represent the steps of the wizard.
+    """
+
+    OUIA_ID = 'bulk-repo-sets-wizard-modal'
+
+    # Hidden alert - Change the status of at least one repository.
+    repo_set_alert_icon = './/div[@class="pf-v5-c-alert__icon"]'
+    repo_set_alert_msg_xpath = (
+        f'{repo_set_alert_icon}/following-sibling::h4[@class="pf-v5-c-alert__title"]'
+    )
+
+    title = './/h2[@class="pf-v5-c-wizard__title-text"]'
+    content_override_action_dropdown = Dropdown(
+        locator='.//button[@data-ouia-component-id="OUIA-Generated-MenuToggle-primary-1"]'
+    )
+
+    close_btn = Button(locator='.//div[@class="pf-v5-c-wizard__close"]')
+    cancel_btn = Button(locator='//button[normalize-space(.)="Cancel"]')
+    back_btn = Button(locator='//button[normalize-space(.)="Back"]')
+    next_btn = Button(locator='//button[normalize-space(.)="Next"]')
```

Comment:
Replace Button with PF5Button, that I am. adding in https://github.com/SatelliteQE/airgun/pull/1814

Reply:
Do at least those one, but definetly you can do more if you check them